WidgetExtension/Variants/Hashtag/HashtagWidgetView.swift

```diff
@@ -81,54 +81,63 @@ struct HashtagWidgetView: View {
                     .fontWeight(.heavy)
                     .foregroundColor(.secondary)
             }
-            Text(statusHTML: entry.hashtag.content, fontSize: 11, fontWeight: 510)
+            Text(statusWithColorHashtags: entry.hashtag.content, fontSize: 11, fontWeight: .medium)
                 .lineLimit(3)
         }
     }
 }
 
 /// Inspired by: https://swiftuirecipes.com/blog/swiftui-text-with-html-via-nsattributedstring
 extension Text {
-    init(statusHTML htmlString: String, fontSize: Int = 16, fontWeight: Int = 400, colorScheme: ColorScheme = .light) {
-
-        let textColor = (UIColor(named: "Colors/TextColor") ?? UIColor.gray).hexValue
-        let accentColor = (UIColor(named: "Colors/Blurple") ?? UIColor.purple).hexValue
-
-        let fullHTML = """
-<!doctype html>
-<html>
-    <head>
-        <style>
-                body {
-                    font-family: -apple-system;
-                    font-size: \(fontSize)px;
-                    font-weight: \(fontWeight);
-                    line-height: 133%;
-                    color: \(textColor);
-                }
+    init(statusWithColorHashtags htmlString: String, fontSize: CGFloat = 16, fontWeight: UIFont.Weight = .regular) {
 
-                a {
-                    color: \(accentColor);
-                }
+        let textColor = UIColor(named: "Colors/TextColor")
+        let accentColor = UIColor(named: "Colors/AccentColor")
+        let font = UIFontMetrics(forTextStyle: .headline).scaledFont(for: .systemFont(ofSize: fontSize, weight: fontWeight))
+
+        let attributedString: AttributedString
+
+        // 1. Render status-content as HTML ...
+        if let data = htmlString.data(using: .utf8),
```

Comment:
💡  Maybe use `guard let …` here to save on one indentation?

Reply:
That's an idea, thank you :)